### ISSUE TITLE
Add compatibility to Level Up: Advanced 5th Edition system for shops

### ISSUE THREAD

**Issue by sivrehn** [ironmonk88/monks-enhanced-journal#560]
Hello,

I am a community moderator for the foundry version of the Level Up: Advanced 5th Edition system
Your module was compatible with our system, but some changes were made to the data structure and the shops no longer work :( .
I am not very familiar with what information you'd need to make it compatible again, if that's feasible....

Here's (attached) what it gives me when I try to make a trade. [Document.txt](https://github.com/ironmonk88/monks-enhanced-journal/files/11513543/Document.txt)

Whatever system information you may require, just ask me and I'll get it from the developers right away. :) I am told no more breaking databank changes will occur - I have waited for a while to submit this request, until all such changes were made, so you wouldn't have to do this more than once.

Thanks again for the wonderful modules and the attention.

Siv.

**Comment by ironmonk88** [comment]
So I fixed a small issue that was preventing the purchase of an item.

But.... you won't be able to drag and drop an item onto the character sheet to purchase it.
The latest A5e doesn't call the `dropActorSheetData` Hook any more, and MEJ requires that to handle asking for quantities.
The other issue would be that the onDrop event on the Character sheet tries to get the item from a Uuid.  Which wouldn't work for MEJ because in Foundry Journal Sheets aren't allowed to store items.  It gets around this by sending all the Item information in a `data` property, which other systems will convert into an Item before using it.

I've fixed it so it will be usable, but can't add all the functionality.
